src/routes/web.php

```diff
@@ -12,3 +13,4 @@
 Route::get('/profile/show', [UserController::class, 'show'])->name('profile.show');
 Route::post('/logout', [UserController::class, 'logout'])->name('user.logout');
 Route::get('/home', [UserController::class, 'home'])->name('user.home');
+Route::post('/post', [PostController::class, 'store'])->name('post.store');
```

Comment:
```suggestion
Route::post('/posts', [PostController::class, 'store'])->name('post.store');
```

Reply:
I have resolved this suggestion, sir. Thank you! 🙇